site/content/blog/simplifying-package-authoring.md

```diff
@@ -0,0 +1,301 @@
+---
+title: "Simplifying Package Authoring"
+slug: simplifying-package-authoring
+date: 2022-08-17
+author: Rohit Aggarwal
+excerpt: "Simplifying Package Authoring with kctrl"
+image: /img/logo.svg
+tags: ['carvel', 'package', 'author', 'kctrl', 'create']
+---
+
+In Carvel, Kubernetes(K8s) manifest are distributed and consumed via the concept of Package. A package Author encapsulates, version, and distribute K8s manifest for package consumers to install on a Kubernetes cluster. A package author can choose to create a package by using third party manifest e.g. they can choose to create a package from cert-manager, dynatrace etc. or they can distribute their own project K8s manifest by creating a Package.
+
+Package Authoring is an iterative process and below are the steps authors generally go through:
+1. Authors know about the K8s manifest they want to package.
+2. Add/change the manifest by adding additional overlay/template and test the package. This is the iterative part where authors want to make the changes and test them quickly.
+3. Once all the manifest are in place, create the imgpkg bundle(to be mentioned in the package) and the package itself.
+4. Add the package to the package repository for distribution.
+
+Today, package authors are supposed to know all the Carvel tools as they are being used in the package authoring journey. Learning the Carvel tools before package authoring has a learning curve off itself. As part of simplifying package authoring, we wanted to reduce this learning curve so that authors can focus on package authoring. As part of this, few commands have been added to the already existing `kctrl` cli.
+
```

Comment:
There needs to be a paragraph along the lines of "In this blog post, I will show you../ I will walk you through... / I will go over.../" something that helps the reader understand what they are about to read and why they would want to keep reading. What are they about to learn???

Reply:
Adopted. `In this blog, we will use the above commands to create a `Dynatrace Operator` Package.`